argo/kfp-compiler/integration/Dockerfile

```diff
@@ -5,6 +5,7 @@ FROM python:$PYTHON_VERSION
 ARG TFX_VERSION
 
 RUN pip install tfx==$TFX_VERSION
+RUN pip install -U google-cloud-aiplatform "shapely<2"
```

Comment:
if the shapely dependency doesn't work on master already, we should raise a separate PR for this

Reply:
If I do that, then that PR will have to be merged first, otherwise this PR will fail?

Reply:
yes, that's what we should do

Reply:
ok, will do that later this week

Reply:
After some digging, I believe that it should be part of this PR.

`shapely` is only a dependency of `google-cloud-aiplatform` of which:
- the latest and not working is `1.38.1`
- `tfx` `1.14` requires `>=1.6.2,<2`
- `apache-beam` requires `>=1.26.0,<2.0` (and this is a dependency of `tfx` BTW)

Any version `>=1.26.0,<1.35.0` would work. This [changed in `tfx` `1.14`](https://github.com/tensorflow/tfx/blob/v1.14.0/RELEASE.md#dependency-updates), that relaxed the version constraints on those dependencies.

Reply:
leaving as-is